chore: upgrade Prettier and run on full repo

nschonni opened the pull request:
\*\*Description:\*\*

\- Move Prettier 1 \->2 to support \`.\` glob and built\-in parsers
\- Move whitespace settings to EditorConfig that Prettier reads
\- Expand glob to run on all non\-generated files

\*\*Related issue:\*\*
Add link to the related issue.

\*\*Check list:\*\*

\- \[ \] Mark if documentation changes are required.
\- \[ \] Mark if tests were added or updated to cover the changes.

nschonni commented:
@oscard0m probably easier to review this with the "whitespace ignore" option

dmitry-shibanov commented:
Hello @nschonni. Thank you for your pull request. We'll investigate it.

lordt4ever commented:
No £

Shutch748 reviewed (changes requested):
Doesn't belong to them

nschonni committed:
chore: upgrade Prettier and run on full repo

- Move Prettier 1 -> to support `.` glob and built-in parsers
- Move whitespace settings to EditorConfig that Prettier reads
- Expand glob to run on all non-generated files
